src/main/java/org/spongepowered/common/mixin/core/block/MixinBlockBanner.java

```diff
@@ -72,4 +74,9 @@ public boolean supports(Class<? extends ImmutableDataManipulator<?, ?>> immutabl
     private ImmutableAttachedData getIsAttachedFor(IBlockState blockState) {
         return ImmutableDataCachingUtil.getManipulator(ImmutableSpongeAttachedData.class, blockState.getBlock() instanceof BlockBannerHanging);
     }
+
+    @Override
+    public Translation getTranslation() {
+        return new SpongeTranslation("item.banner.white.name");
```

Comment:
Do we need the translations to be re-created every time?

Reply:
Are they worth caching ?
Currently there is no "standard way" of doing the translation. Half of the blocks cache the translation (ex. https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/mixin/core/data/types/MixinBlockPrismarineEnumType.java )
Either way, I will probably fix all of them to be consistent in the next PR. (so, input wanted)

Reply:
AFAICT either the translations are not used at all or used multiple times (because a plugin actually wants them)

IMO lazy caching is a good way that does not consume that much memory.